src/sentry/templates/sentry/emails/activity/new-user-feedback.txt

```diff
@@ -4,8 +4,14 @@
 
 {{ report.name }} left a new comment:
```

Comment:
Is it ok to display the reporter's name if enhanced privacy is enabled?

Reply:
Probably our call. The company requesting this said they're fine with the username -- only concerned about the content of the feedback being hidden.

Reply:
It's probably fine for now then, maybe a product decision.